### ISSUE TITLE
Release Notes

### ISSUE THREAD

**Issue by togethere** [joomla/joomla-developer-docs#42]
Hi everyone,

IMHO it is necessary to have release notes for every published release (major/minor/batch). I missed that and when I asked the community I was told to look up the repository. This was done by many of us, everyone on itself.

To align that into a process I have a volunteer who offers his help to contribute with looking up the repo and provide a summary. This has obviously to be finally checked before a release is published. 

I think it could be helpful for the project itself (Marketing, Docs, Certification etc.) and for those using Joomla! as a tool - to be aware of what has changed.

My question is:

Which team is responsible to implement release notes and not keep them secret? PLT???
How can we ensure to have the right release notes every time (sometimes the code is changing and some parts are revoked from the release)? 

What do you think about having official release notes implemented into the release process?

**Comment by coolcat-creations** [comment]
Yes, would be great to have a summary / a changelog. Is that possible @wilsonge ?

**Comment by mbabker** [comment]
Can this be moved to an "active" repo?

We stopped doing a "changelog" file years ago because it really wasn't a changelog; it was in essence those of us with commit rights adding the commit message and a couple arbitrary symbols to a new line in a text file; for the most part that "changelog" file was the commit log.  This was phased out in part to make our lives as the people with commit rights easier (to merge a pull request we had to pull down the diff for it, manually apply it to a local checkout, change that file, and remember to commit the changes with the correct configuration values to keep things like authorship correctly attributed; now we just click the merge button).

We have always provided a list of items closed during a release period.  It used to be a page like https://developer.joomla.org/version-3-3-0-release-notes.html which corresponded to a list of JoomlaCode issues, now it's the link to the list of closed issues grouped under the GitHub milestone.  This will always be the most accurate snapshot of all of the activity and IMO no amount of human activity would make this better.

That's not to discourage someone from wanting to include a high level summary in the release announcements.  But really at that point you're starting to make arbitrary decisions about what items are important enough to be included in that list.  Without making a list of guidelines for that, you can have 3 or 4 different people sit down and go through a milestone's issue log and come up with 10-15 items individually and at best 2 or 3 would overlap from each list.

**Comment by coolcat-creations** [comment]
I understand it´s not easy, but maybe each item that requires documentation changes can go into the list. Does this make it easier? Where and how to move this? Just open a new one under ... ? Thanks @mbabker!

**Comment by mbabker** [comment]
Open a new item somewhere other than this dead repo, it's pretty easy to decipher which ones are active 😉

**Comment by brianteeman** [comment]
The answer is 
https://github.com/joomla/joomla-cms/commits/3.6.4

iirc this is on the release announcement
its also on the readme

**Comment by brianteeman** [comment]
Can you please raise this on an active repo
either
https://github.com/joomla/joomla-websites/issues
or 
https://github.com/joomla/joomla-cms/issues
